organisaatio-service/src/main/java/fi/vm/sade/organisaatio/business/impl/OrganisaatioBusinessServiceImpl.java

```diff
@@ -752,7 +753,7 @@ public void updateOrganisaatioNameHierarchy(Organisaatio oppilaitos, Map<String,
     }
 
     @Override
-    public OrganisaatioNimi newOrganisaatioNimi(String oid, OrganisaatioNimiDTOV2 nimidto) {
+    public OrganisaatioNimi newOrganisaatioNimi(String oid, OrganisaatioNimiDTO nimidto) throws OrganisaatioModifiedException {
```

Comment:
should this be a checked exception or skip declaring it?

Reply:
i'll remove declaration, since i think its a mistake.